types/react-expand-animated/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for react-expand-animated 1.0
+// Project: https://github.com/sonybinhle/react-expand-animated
+// Definitions by: Ben Langlois <https://github.com/BenLanglois>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+export interface ExpandProps {
+    /**
+     * The expanded or collapsed content
+     */
+    children: React.ReactNode;
+
+    /**
+     * When set to true expand the children component otherwise collapse it
+     */
+    open?: boolean;
+
+    /**
+     * Animation duration in ms
+     */
+    duration?: number;
+
+    /**
+     * Css3 Animation's type
+     */
+    easing?: string;
```

Comment:
Maybe you want to use something like from csstype like https://github.com/frenic/csstype/blob/6efe9255f529786fbe37405fc608fb885eef67e1/index.js.flow#L6084? (csstype is the library DefintelyTyped uses for React under the hood for its CSS types)

Reply:
this was wrong I believe, read my comments, thx!